astropy/coordinates/erfa_astrom.py

```diff
@@ -213,13 +254,20 @@ def apci(self, frame_or_coord):
             For this function, a CIRS frame is expected.
         '''
         obstime = frame_or_coord.obstime
+        # no point in interpolating for a single value
         support = self._get_support_points(obstime)
 
-        cip = self._get_cip(support, obstime)
```

Comment:
I think this means that the static method `_get_cip` is no longer used...

Reply:
it has come back again as a result of adding `apco`